ports/mimxrt/hal/flexspi_hyper_flash.c

```diff
@@ -44,17 +45,17 @@ __attribute__((always_inline)) static inline void clock_disable_clock(clock_ip_n
     clock_control_gate(name, kCLOCK_ClockNotNeeded);
 }
 
-#define DIV_PAGE_PGM	4
-#define DIV_ERASE_PGM	4
-#define DIV_READ		0
+#define DIV_PAGE_PGM    4
+#define DIV_ERASE_PGM   4
+#define DIV_READ                0
```

Comment:
please align these values to the same column

Reply:
fixed.